blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/analytics/AbstractAnalytics.java

```diff
@@ -42,6 +47,11 @@ public void track(TrackRequest req) {
         if (StringUtils.isEmpty(req.name)) {
             throw new ServiceException.BadRequestException("missing name");
         }
+        String server = server();
+        AnalyticsConfig config = AnalyticsConfigService.get();
+        if (!belongsToActiveCohort(config, server)) {
+            LOGGER.log(Level.FINE, "Server is not in active cohort. No analytics event has been sent");
```

Comment:
@i386 Looks like when `! belongsToActiveCohort()` is true then it should skip sending events. Missing return here?